v2/api/authorization/v1alpha1api20200801preview/role_assignment_types_gen.go

```diff
@@ -68,7 +68,7 @@ func (assignment *RoleAssignment) ConvertFrom(hub conversion.Hub) error {
 func (assignment *RoleAssignment) ConvertTo(hub conversion.Hub) error {
 	// intermediate variable for conversion
 	var destination alpha20200801ps.RoleAssignment
-	err := assignment.AssignPropertiesToRoleAssignment(&destination)
+	err := assignment.AssignProperties_To_RoleAssignment(&destination)
```

Comment:
Was this change part of Georges changes too?

Reply:
No, but a change that made sense, driven by generated identifiers like:

```
AssignPropertiesToFrontendIPConfiguration_STATUS_LoadBalancer_SubResourceEmbedded
```